frontend/src/v5/ui/routes/dashboard/projects/projectsList/projectListItem/projectListItem.component.tsx

```diff
@@ -20,22 +20,52 @@ import {
 	DashboardListItemRow,
 	DashboardListItemText,
 } from '@components/dashboard/dashboardList/dashboardListItem/components';
-
+import {
+	ShareModalProject as ShareModal,
+} from '@components/dashboard/dashboardList/dashboardListItem/shareModal/shareModalProject/shareModalProject.component';
+import { useState } from 'react';
 import { discardSlash } from '@/v5/services/routing/routing';
+import { formatMessage } from '@/v5/services/intl';
+import { IProject } from '@/v5/store/projects/projects.types';
+import { Button } from '@controls/button';
+import { FormattedMessage } from 'react-intl';
+
+type ProjectListItemProps = {
+	project: IProject,
+};
 
-export const ProjectListItem = ({ projectId, name }): JSX.Element => {
+export const ProjectListItem = ({ project }: ProjectListItemProps): JSX.Element => {
 	let { url } = useRouteMatch();
 	url = discardSlash(url);
 
+	const [openModal, setOpenModal] = useState(false);
```

Comment:
As there will be other modals here it'd probably best to specify which modal this state refers to